shopfloor/tests/test_actions_data_detail.py

```diff
@@ -91,6 +90,10 @@ def test_data_package(self):
             },
         }
         self.assertDictEqual(data, expected)
+        data = self.data_detail.package_detail(
+            package, picking=self.picking, with_package_move_line_count=True
+        )
+        expected.update({"move_line_count": 2})
```

Comment:
I don't understand this change with no assertion added after ?

Reply:
oh yes this was missing. Thx